Makefile

```diff
@@ -1,18 +1,56 @@
 GOTOOLS = \
-					github.com/mitchellh/gox \
-					github.com/Masterminds/glide \
-					gopkg.in/alecthomas/gometalinter.v2 \
-					github.com/gogo/protobuf/protoc-gen-gogo \
-					github.com/gogo/protobuf/gogoproto
-
+	github.com/mitchellh/gox \
+	github.com/Masterminds/glide \
+	gopkg.in/alecthomas/gometalinter.v2 \
+	github.com/gogo/protobuf/protoc-gen-gogo \
+	github.com/gogo/protobuf/gogoproto
+GOTOOLS_CHECK = gox glide gometalinter.v2 protoc protoc-gen-gogo
+PACKAGES=$(shell go list ./... | grep -v '/vendor/')
 INCLUDE = -I=. -I=${GOPATH}/src -I=${GOPATH}/src/github.com/gogo/protobuf/protobuf
 
-all: protoc install test metalinter
+all: check protoc build test install metalinter
 
-PACKAGES=$(shell go list ./... | grep -v '/vendor/')
+check: check_tools get_vendor_deps
```

Comment:
Opening comment says `Check checks stuff without making side effects.`. So this probably shouldn't have `get_vendor_deps`

Reply:
Yeah.  We can do:
```
all: check get_vendor_deps protoc build test install metalinter
check: check_tools
```